_posts/en/newsletters/2019-07-31-newsletter.md

```diff
@@ -0,0 +1,241 @@
+      than 100 MB a month.  Clients would also need to download matching
+      blocks, which would vary depending on their activity (roughly one
+      block for every sent or received transaction for typical users; most
+      blocks are less than 2 MB in size).  It's unclear how many users
+      would be significantly affected by having to download that amount
+      of data.
+
+    - *Interested parties can run their own nodes:* the change to
+      Bitcoin Core only disables serving bloom filters by default.  It
+      doesn't remove the feature.  Authors of wallets depending on bloom
+      filter support can easily operate their own nodes, and they can
+      also try to persuade other node operators who aren't worried about
+      the DoS vulnerability to set the configuration option to true,
+      e.g.  `peerbloomfilters=1`.
+
+## Bech32 sending support
+
+*Week 20 of 24 in a [series][bech32 series] about allowing the people
+you pay to access all of segwit's benefits.*
+
+{% include specials/bech32/20-percentage-loss.md %}
+
+## Selected Q&A from Bitcoin StackExchange
+
+*[Bitcoin StackExchange][bitcoin.se] is one of the first places Optech
+contributors look for answers to their questions---or when we have a
+few spare moments of time to help curious or confused users.  In
+this monthly feature, we highlight some of the top voted questions and
+answers made since our last update.*
+
+{% comment %}<!-- https://bitcoin.stackexchange.com/search?tab=votes&q=created%3a1m..%20is%3aanswer -->{% endcomment %}
+{% assign bse = "https://bitcoin.stackexchange.com/a/" %}
+
+- [Why does the `importmulti` RPC not support zpub and ypub?]({{bse}}89261) Fontaine
+  asks why Bitcoin Core's `importmulti` RPC supports the xpub public key
+  derivation format but not ypub or zpub formats. Pieter Wuille explains that
+  the xpub format was specified in BIP32 and was used to generate
+  P2PKH addresses types which were the most common at the time. Pieter goes on to
+  describe Bitcoin Core's [output script descriptors][descriptor] as a more sustainable
+  approach to address generation.
+
+- [Is Bitcoin PoW actually SHA256 + Merkle tree generation?]({{bse}}89296) User ascendzor
+  wonders about the consequences of miners needing to recompute the merkle root
+  as part of proof of work mining and why the nonce cannot be increased to 64
+  bits. Pieter Wuille explains that not only is the overhead in computing the
+  merkle root negligible, but an increase in the nonce size would require a hard
+  fork.
+
+- [What is the difference between bytes and virtual bytes
+  (vbytes)?]({{bse}}89385) Ugam Kamat and Murch note the differences between
+  virtual size (vsize, denominated in vbytes) and size (denominated in bytes)
+  and go on to explain the block weight limit and segwit discount.
+
+- [To what extent does asymmetric cryptography secure bitcoin
+  transactions?]({{bse}}89262) RedGrittyBrick explains that
```

Comment:
I think sipa's answer is more succinct and clearer here.

Reply:
Agreed and updated. I was trying to prevent SE section from becoming sipa show :), but it is a good answer.